trezorlib/tests/unit_tests/test_stellar.py

```diff
@@ -0,0 +1,313 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2018 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import base64
+from trezorlib import stellar, messages as proto
```

Comment:
please split into two lines:
```python
from trezorlib import messages
from trezorlib import stellar
```

also please change all usages of `proto` to `messages`. (ideally in stellar.py too)